app/commands/solution/publish_iteration.rb

```diff
@@ -10,9 +10,15 @@ def call
     Solution::UpdatePublishedExerciseRepresentation.(solution)
     Solution::UpdateSnippet.(solution)
     Solution::UpdateNumLoc.(solution)
+    Infrastructure::InvalidateCloudfrontItems.(
+      :website,
+      ["/tracks/#{track.slug}/exercises/#{exercise.slug}/solutions/#{user.handle}.jpg"]
+    )
```

Comment:
Maybe extract the file path formatting to a separate method? e.g. on `Solution`?